packages/contracts/contracts/universal/ISimpleMultiSigWallet.sol

```diff
@@ -0,0 +1,70 @@
+// SPDX-License-Identifier: MIT
+pragma solidity 0.8.15;
+
+/**
+ * @title ISimpleMultiSigWallet
+ * @notice Interface for contracts of a simple multisig wallet.
+ */
+interface ISimpleMultiSigWallet {
```

Comment:
I've tried to keep it to the basic interface that our clients currently use, but there are things that need to be considered in each case, so I think it's best to put them in when they're needed.

Reply:
I got your point, but I think these are needed for now. For example, even if our client is not using `revokeConfirmation` tx, this should be implemented because in some exceptional case we may need to revoke the confirmation manually.
For current contract, nobody can revoke their confirmation. It is quite weird to me. 

And it seems that we don't need `executionFailed` event because our `executeTransaction` is implemented so that the tx must succeed.

Reply:
`getTransactionCount`, `getConfirmations` is still missing.